src/video/video_driver.hpp

```diff
@@ -273,6 +273,16 @@ class VideoDriver : public Driver {
 	 */
 	virtual bool PollEvent() { return false; };
 
+	/** Gamepad support */
+
+	/**
+	* Open the gamepad for input.
+	* @return True if the gamepad is available and open. False if no gamepad is available.
+	*/
```

Comment:
Missing one space indentation before *
```suggestion
	/**
	 * Open the gamepad for input.
	 * @return True if the gamepad is available and open. False if no gamepad is available.
	 */
```